settings.py

```diff
@@ -13,7 +13,7 @@
 SDN_TRACE_CP_URL = "http://localhost:8181/api/amlight/sdntrace_cp"
 
 # EVC consistency interval
-DEPLOY_EVCS_INTERVAL = 60
+DEPLOY_EVCS_INTERVAL = 5
```

Comment:
```suggestion
DEPLOY_EVCS_INTERVAL = 60
```

Gretel, I believe you changed the interval incorrectly, no?